mapfishapp/src/main/webapp/app/js/GEOR.js

```diff
@@ -410,7 +410,28 @@ Ext.namespace("GEOR");
         // Note: we're providing GEOR.ajaxglobal.init as a callback, so that
         // errors when loading WMC are not catched by GEOR.ajaxglobal
         // but by the mapinit module, which handles them more appropriately
-
+        if (GEOR.mapinit) {
+            GEOR.mapinit.events.on({
+                "searchresults": function(options) {
+                    removeActiveTab();
+                    var tab = new GEOR.ResultsPanel({
+                        html: tr("resultspanel.emptytext"),
+                        tabTip: options.tooltip,
+                        title: options.title,
+                        map: map
+                    });
+                    tab.populate({
+                        features: options.features,
+                        // here we do have a valid model (got from describeFeatureType)
+                        model: options.model
+                    });
+                    southPanel.insert(southPanel.items.length-1, tab);
+                    southPanel.setActiveTab(tab);
+                    southPanel.expand();
+                    tab._zoomToFeatures(options.features);
```

Comment:
In order to zoom on a set of features, the method is the same a the one in _zoomToFeatures : calculate total bounds of features or get dataExtent and zoom on it.

Maybe we can set this method as a public one ?